index.html

```diff
@@ -0,0 +1,158 @@
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"/>
+  <link rel="stylesheet" href="styles.css" type="text/css">
+  <link href="https://fonts.cdnfonts.com/css/tasty-food" rel="stylesheet">
+</head>
+<body>
+
+
+    <section class="intro-container"> 
+        <header>
+        <nav>
+          <div class="mobile hamburger-icon">
+            <div class="drop-menu hidden-menu"> 
+                <div class="close-button">X</div>
+                <ul class="nav-menu">
+                    <li class="nav-menu-item"><a href="./about.html" class="nav-link">About</a></li>
+                    <li class="nav-menu-item"><a href="#main-program" class="nav-link">Program</a></li>
+                    <li class="nav-menu-item"><a href="#featured-chefs" class="nav-link">Presenters</a></li>
+                    <li class="nav-menu-item"><a href="#partners" class="nav-link">Partners</a></li>
+                </ul>    
+            </div>
+            <i class="fas fa-bars"></i>
+          </div>
+        </nav>
+        <div class="mobile nav-line"></div>
+        <div class="top-bar">
+            <ul>
+                <li><a href="#"><img class="social-icon" src="./images/facebook.png"></a></li>
+                <li><a href="https://www.facebook.com/mchafu"><img class="social-icon" src="./images/twitter.png"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="https://thetjaypod.wordpress.com/">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+
+            </ul>
+        </div>
+        <div class="desk-nav">
+            <img class="logo-home" src="./images/Shakys-logo.jpeg">
+            <ul class="desk-nav-menu">
+                <li class="desk-nav-menu-item"><a href="./about.html" class="desk-nav-link">About</a></li>
+                <li class="desk-nav-menu-item"><a href="#main-program" class="desk-nav-link">Program</a></li>
+                <li class="desk-nav-menu-item"><a href="#featured-chefs" class="desk-nav-link">Presenters</a></li>
+                <li class="desk-nav-menu-item"><a href="#partners" class="desk-nav-link">Partners</a></li>
+            </ul>
+            <div class="register-now">Register Now</div>
+        </div>
+
+      </header>
+<div class="welcome-container">
+    <div class="headings">
+        <h2 class="quote">"Good Food Nourishes Souls."</h2>
```

Comment:
- [ ] kindly increase the padding between the nav and first text (Good Food Nourishes Souls)
- [ ] please, the text is aligned with the nav bar logo. Kindly try as much as possible to follow the design throughout
- [ ] please make the date bolder like in the original design
- [ ] add some padding to the date, title and the text in the box. please try to follow the original design as much as possible
- please make the necessary changes to the mobile version as well
yours
![Screenshot (804)_LI](https://user-images.githubusercontent.com/65298048/219615747-7f0f2860-0be5-456f-8fa5-0ac0f829e08d.jpg)
original
![Screenshot (807)](https://user-images.githubusercontent.com/65298048/219616473-08347c39-9b2a-4288-8b06-ae83831c0c10.png)